blog/2025-03-05-open-invite-vendor-commitee.md

```diff
@@ -0,0 +1,79 @@
+---
+slug: "open-invite-vendor-commitee"
+title: "Open Invitation to the Vendor Commitee"
+date: 2025-03-05
+authors: [weyert]
+description: "Open Invitation to the Vendor Commitee"
+tags: [cncf, vendors, community]
+draft: false
+---
+
+# Join OpenFeature's Vendor Council: Shaping the Future of Feature Flagging
+
+![OpenFeature Logo](https://openfeature.dev/img/logos/open-feature-horizontal-color.svg)
+
+## An Open Invitation to Feature Flagging Vendors and Maintainers
```

Comment:
I think you can remove this header too and start right off with the paragraph below.